backend/requirements/base.txt

```diff
@@ -9,20 +9,20 @@ chardet==3.0.4
 Django==1.11.29
 django-appconf==1.0.2
 django-compressor==2.1.1
-django-environ==0.11.2
+django-environ==0.10.0
 django-extensions==2.1.0
 django-filter==1.0.4
 django-simple-history==1.9.0
 django-storages==1.7.2
-djangorestframework==3.4.6
-djangorestframework-expiring-authtoken==0.1.4
```

Comment:
This was originally 0.1.4.  Does it need rolled back further?

Reply:
The version installed before moving to CDK was 0.1.4. However, pip wasn't finding it, so I moved it to 0.1.3. The documentation for the package is pretty sparse...